tests/virt_autotest/libvirt_virtual_network_init.pm

```diff
@@ -37,6 +37,10 @@ use version_utils 'is_sle';
 sub run_test {
     my ($self) = @_;
 
+    #Need to reset up environemt - br123 for virt_atuo test due to after
+    #finished guest installation to trigger cleanup step on sles11sp4 vm hosts
+    virt_autotest::virtual_network_utils::restore_standalone() if (is_sle('=11-sp4'));
+
```

Comment:
Generally speaking, please add verification run to cover bother 11-sp4 and non-11-sp4.

Reply:
And there are too much is_sle(=11-sp4), please make sure you organize them well enough.

Reply:
Good point. 
Will updated this as your mentioned soon.

Reply:
Had reorganized is_sle(=11-sp4) here already.